polaris-react/src/components/IndexTable/IndexTable.scss

```
@@ -994,28 +843,15 @@ $loading-panel-height: 53px;
 
 .SelectAllActionsWrapper {
   visibility: visible;
-  position: relative;
+  position: static;
   // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
   z-index: var(--pc-index-table-bulk-actions);
-  top: 0;
-  left: 0;
-  right: 0;
-  padding: 0 var(--p-space-2) 0 var(--p-space-4);
   background: var(--p-color-bg);
-
-  // TODO: Remove this; condensed doesn't have bulk actions / select all
-  &.StickyTableHeader-condensed {
-    padding-top: calc(var(--p-space-2) + var(--p-space-05));
-  }
-
-  #{$se23} & {
-    // Note: this undoes top/left/right positioning from pre-se23
-    position: static;
-    padding-left: var(--p-space-3);
-    padding-top: var(--p-space-1_5-experimental);
-    padding-bottom: var(--p-space-1_5-experimental);
-    line-height: var(--p-font-line-height-2);
-  }
+  padding-left: var(--p-space-3);
+  padding-right: var(--p-space-2);
+  padding-top: var(--p-space-1_5-experimental);
+  padding-bottom: var(--p-space-1_5-experimental);
```

Being pedantic here, and this will need to be auto-formatted (line is too long), but...

```suggestion
  padding: var(--p-space-1_5-experimental) var(--p-space-2) var(--p-space-1_5-experimental) var(--p-space-3);
```

---

I'm going to leave this out, given how long the variable names are, I'd prefer the explicit multi-line invocation, rather than shorthand.